data/korath/korath missions.txt

```diff
@@ -439,7 +439,7 @@ mission "Efreti: Distress Call 5"
 	destination "Greenwater"
 	landing
 	passengers 66
-	blocked `You need <bunks> available to carry the salvage team.`
+	blocked `You will need <bunks> available bunks to carry the salvage team.`
```

Comment:
```suggestion
	blocked `You need <bunks> available bunks to carry the salvage team.`
```
I think in most cases we just say "you need blank". It's odd to say you "will" need bunks when you're already at the planet to do the job.

Reply:
Except for in this mission you aren't actually carrying anybody; when you complete the mission you pick up the passengers.

Reply:
The mission still requires you have 66 passengers. While the future mission requires them, this one does to, therefore this isn't just about "will need", it's about needing the space now.